scss/bitstyles/settings/shadows.stories.mdx

```diff
@@ -0,0 +1,145 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Design Tokens/Shadows" />
+
+# Shadows
+
+Bitstyles provides a global list of shadow specifications, to maintain consistency of shadows throughout your components. The shadows are specified in a sass list so they can be translated to various formats as needed — the order of parameters for CSS `box-shadow` is different to the CSS filter `drop-shadow()`. We need to be able to translate to either of those formats.
+
+The default shadows used throughout bitstyles are multi-layered to create a more natural or realistic appearance than you get from a single box-shadow or filter. We provide a generator function that you can use to produce “natural” multi-layered shadows in the correct format for our Sass design system, and functions to translate those to useful formats.
+
+## Generating shadows
+
+The simplest way to override the global shadow design tokens in `$bitstyles-shadows-shadows` is to use `shadow.generate()`:
+
+```scss
+@use '../tools/shadow';
+
+$bitstyles-shadows-shadows: (
+  'default': shadow.generate(
+    $color: rgba(palette.get('brand-1'), 0.15),
+    $offset: 2rem,
+    $blur-radius: 2rem,
+  ),
+);
+```
+
+This creates a specification for a multi-layered shadow using the `brand-1` color with a `2rem` blur, offset `2rem` downwards, which looks like the following examples.
+
+<Canvas>
+  <Story id="utilities-shadow--u-shadow-default" />
+  <Story id="utilities-drop-shadow--u-drop-shadow-default" />
+</Canvas>
+
+### Parameters
+
+|Name|Format|Description|
```

Comment:
Cosmetic derailing, but I personally prefer Markdown tables with whitespace, for the sake of readability in code.

See [this comment](https://github.com/prettier/prettier/issues/5651#issue-391680593) for how they look.

From what I remember, running those files here through prettier, you'd also get that.

Reply:
thanks, I’ll take a look 🙏 these were all run through prettier tho — I’ll look into that too, check why nothing was flagged or changed

Reply:
Ah I was wrong — the linting (& prettier) is over on #609. As that PR will be merged later, I’ll leave this change until then

Reply:
Cool. Curious to see what prettier does to `.mdx` files.

Reply:
It handled it well!  Added line breaks after headings etc, seemed to ignore the JSX ✨